content/integrations/appetize-integration.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Appetize integration
+description: How to integrate your workflows with Appetize using codemagic.yaml
+weight: 11
+---
+
+[**Appetize**](https://appetize.io/) enables you to run native iOS and Android mobile apps directly in your browser. No downloads, plugins, or extra permissions needed. 
+
+## Getting Started
+Before getting started you need to get some tokens and keys.
+
+### Get Appetize Token
+All users with admin or developer roles may request an API token after logging in at [account](https://appetize.io/account) page.
+After getting your **API_TOKEN** you need to add it to your [environment variables](/variables/environment-variable-groups/#storing-sensitive-valuesfiles) in a group named **appetize** for example.
+
+### Get Your App PublicKey
+You have at least to upload your app once to [Appetize](https://appetize.io/upload) manually and then you can get the app *publicKey*.
+in your *codemagic.yaml* file add the app key as a variable.
+
+{{< highlight yaml "style=paraiso-dark">}}
+    environment:
+      groups:
+        - appetize # <-- (Includes API_TOKEN)
+      vars:
+        APP_PUBLIC_KEY: "YOUR_APP_PUBLIC_KEY"
```

Comment:
Why set this locally in the YAML when it could be added to the `appetize` group that contains the API token?